modules/apps/subscription/subscription-test-util/src/main/java/com/liferay/subscription/test/util/BaseSubscriptionLocalizedContentTestCase.java

```diff
@@ -55,13 +54,13 @@ public void tearDown() throws Exception {
 	public void testSubscriptionLocalizedContentWhenAddingBaseModel()
 		throws Exception {
 
-		Map<Locale, String> previousLocalizedContents = HashMapBuilder.putAll(
```

Comment:
@marcogalluzzi, isn't there a way to create the test without changing the base clases? They are used in too many places

Reply:
I've tested everywhere it is used. It shouldn't be any problem. The current implementation using the attribute `_localizedContent` seems to be unnecessary, this is why I'm proposing the change as an improvement.